report.txt

```diff
@@ -0,0 +1,83 @@
+
+
+Report
+======
+117 statements analysed.
+
+Statistics by type
+------------------
```

Comment:
I see that this file is in the gitignore but you are still merging this to main? Also is this automatically generated with dslint?

Reply:
I deleted it